api/DragHandler.md

```diff
@@ -0,0 +1,26 @@
+[API categories](API-categories.md) | [API index](API-index.md)
+
+
+# DragHandler (interface)
+
+Implement this interface to handle events related to dragging. The methods of this class will be called on the UI thread.
+
+
+Table of contents:
+* [Callbacks](#callbacks)
+  * [OnDragEnter](#ondragenter)
+
+## Callbacks
+
+
+### OnDragEnter
+
+| Parameter | Type |
+| --- | --- |
+| browser | [Browser](Browser.md) |
+| dragData | [DragData](DragData.md) |
+| mask | int |
+| __Return__ | bool |
+
+ Called when an external drag event enters the browser window.
```

Comment:
This description is not enough. Please copy complete description from CEF headers using template like: "Description from upstream CEF: <quote>....</quote>" - similarly to how it is done in API docs for some of other callbacks.